meta-xt-prod-devel-rcar-control/recipes-guests/domu/files/domu-vdevices-nogsx.cfg

```diff
@@ -0,0 +1,24 @@
+# =====================================================================
+# DomU virtual devices
+# =====================================================================
+
+# Note1:
+# In case of using virtio-disk there should be additional properties:
+# "backendtype=standalone, specification=virtio"
+#
+# Note2:
+# Option `target` must be last element in list. For explanation see
+# https://xenbits.xenproject.org/docs/4.15-testing/man/xl-disk-configuration.5.html
+disk = [ 'backend=DomD, vdev=xvda1, target=/dev/STORAGE_PART3' ]
```

Comment:
... approach described in the other command will ensure that there will be duplicate entries like this.